spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/Binding.java

```diff
@@ -26,9 +28,30 @@
  * @author Mark Fisher
  * @author Gary Russell
  * @author Marius Bogoevici
+ * @author Oleg Zhurakousky
+ * 
  * @see org.springframework.cloud.stream.annotation.EnableBinding
  */
 public interface Binding<T> {
+	
+	/**
+	 * Binds the target component represented by this instance.
```

Comment:
The JavaDocs must be in an an imperative style: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting. Also no blank lines in the method JavaDocs.